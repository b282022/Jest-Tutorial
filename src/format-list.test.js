import formatList from './format-list'

test('Can format a list', () => {
  const formattedList = formatList(
    'Star wars name',
    [
      {name: "Qui-Gon Jinn"},
      {name: "Chewbacca"},
      {name: "Han Solo"},
      {name: "Luke Skywalker"},
    ],
    'name'
  )
  expect(formattedList).toMatchSnapshot()
})
