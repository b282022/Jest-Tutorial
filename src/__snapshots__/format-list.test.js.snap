// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Can format a list 1`] = `
"These are the items in the Star wars name:
 - Qui-Gon Jinn
 - Chewbacca
 - Han Solo
 - Luke Skywalker"
`;
